src/EditorFeatures/CSharpTest/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -1508,6 +1508,27 @@ public void Caller()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseLocalFunction)]
+        [WorkItem(22672, "https://github.com/dotnet/roslyn/issues/24344")]
```

Comment:
@sharwell The work item number is wrong here (22672 instead of 24344). I think I can't correct it because you changed the target branch and if I push it will bring other changes with it. Can you correct this or should we leave it?

Reply:
Sure I can fix it thanks :)